pkg/util/oci/config.go

```diff
@@ -113,7 +113,7 @@ func GetConfig() (*OciConfig, error) {
 
 	var ret *OciConfig
 	for _, o := range sections {
-		if o.Name == "DEFAULT" {
+		if o != nil && o.Name == "DEFAULT" {
```

Comment:
A better fix would be to update parseOciConfig to return an empty list rather than a length one list with a nil pointer when there is no oci config file.

Reply:
Done